Source/Scene/Model/Model3DTileContent.js

```diff
@@ -240,6 +240,7 @@ Model3DTileContent.prototype.update = function (tileset, frameState) {
     model._clippingPlanes !== tilesetClippingPlanes
   ) {
     model._clippingPlanes = tilesetClippingPlanes;
+    model._clippingPlanesState = 0;
```

Comment:
Can we add a unit test to Model3DTileContent, under the "clipping planes" section, for replacing a clipping planes collection with one of the same length?